Alignment/RPData/LHC/2015_10_18_fill4511/version2/sr+el/45.xml

```diff
@@ -0,0 +1,115 @@
+<!--
```

Comment:
@ianna Is the "+" just diff's way of saying that the line was added, i.e. it is not really part of the file?